### ISSUE TITLE
[TF2] Taunt Props stay in their original slot on other player's clients after changing them in loadout menu

### ISSUE THREAD

**Issue by Walka2138** [ValveSoftware/Source-1-Games#4447]
A taunt's prop seems to be assigned to a specific slot when a player first connects to a server, changing the taunts will cause the wrong props to load for other players on the server.
![image](https://user-images.githubusercontent.com/109564743/197682323-71a4a9c2-9b98-4054-b18d-8c4c4e93ae36.png)
for example, if the Surgeon's Squeezebox is moved to the Taunt 1 slot **after** connecting to a server, the taunt will continue to use the Mannbulance's prop. This is _only_ visible from the clients of other players, and taunts will look normal to the player using them
![image](https://user-images.githubusercontent.com/109564743/197682575-a013a3ae-4647-4c2a-b624-4fe7efe4968d.png)
This happens to almost any taunt with a prop, and if the first slot is occupied by a taunt that does not have a visible prop, it will use the player's weapon instead. This can be fixed by simply leaving and rejoining the server, but that is typically an inconvenience, especially for players with multiple taunts.

**Comment by Walka2138** [comment]
Below are a few more examples of the bizarre ways taunts break due to this bug
![image](https://user-images.githubusercontent.com/109564743/197683357-e93fed9e-9efe-4d72-9214-3769620bf2c8.png)
![image](https://user-images.githubusercontent.com/109564743/197683374-37c88526-7d4f-4552-a2b4-5212f0abc9e5.png)
![image](https://user-images.githubusercontent.com/109564743/197683400-8bcdca38-7c79-48d1-bb68-4dbf92bb7b30.png)

**Comment by condor00fr** [comment]
Finally someone talks about this issue. It's been in the game for as long as I can remember and it needs to go away.

**Comment by Thespikedballofdoom** [comment]
I don't know if this fixes every bug that happens with taunts, but any fixes with them I very much hope for too.

**Comment by condor00fr** [comment]
yes, finally I we won't be seing a Pyro dancing the conga while holding his flametrower in is left hand
